CalculatorWithTests.Test/UnitTest1.cs

```diff
@@ -0,0 +1,127 @@
+using System;
+using Xunit;
+
+namespace Calculator.Test
+{
+    public class UnitTest1
+    {
+        [Theory]
+        [InlineData(3, 5, 8)]
+        [InlineData(-3, 5, 2)]
+        [InlineData(0, 0, 0)]
+        [InlineData(-2, -5, -7)]
+        [InlineData(400, 1000, 1400)]
+        [InlineData(-5, 5, 0)]
+        [InlineData(0, 1, 1)]
+        public void AddTogether(int a, int b, int expected)
+        {
+            //Arrange
+            Calculator calculator = new Calculator();
+
+            //Act
+            int actual = calculator.AddTogether(a, b);
+
+            //Assert
+            Assert.Equal(expected, actual);
+        }
+
+        [Theory]
+        [InlineData(5, 3, 2)]
+        [InlineData(3, 4, -1)]
+        [InlineData(432, 31, 401)]
+        [InlineData(-4, -30, -34)]
+        [InlineData(3, 3, 0)]
+        [InlineData(0, 3, -3)]
+        [InlineData(3, 0, 3)]
+        public void Subtract(int a, int b, int expected)
+        {
+            //Arrange
+            Calculator calculator = new Calculator();
+
+            //Act
+            int actual = calculator.Subtract(a, b);
+
+            //Assert
+            Assert.Equal(expected, actual);
+        }
+
+        [Theory]
+        [InlineData(30, 5, 150)]
+        [InlineData(2, 2, 4)]
+        [InlineData(1, 5, 5)]
+        [InlineData(0, 5, 0)]
+        [InlineData(-3, 5, -15)]
+        [InlineData(-4, -5, 20)]
+        [InlineData(-5, 0, 0)]
+        public void Multiply(int a, int b, int expected)
+        {
+            //Arrange
+            Calculator calculator = new Calculator();
+
+            //Act
+            int actual = calculator.Multiply(a, b);
+
+            //Assert
+            Assert.Equal(expected, actual);
+        }
+
+        [Theory]
+        [InlineData(400, 50, 8)]
+        [InlineData(5, 1, 5)]
+        [InlineData(13, 13, 1)]
+        [InlineData(-10, 5, -2)]
+        [InlineData(-8, -2, 4)]
+        [InlineData(40, 5, 8)]
+        [InlineData(0, 5, 0)]
+        public void Divide(int a, int b, int expected)
```

Comment:
There's a suspiciously absent test case here, which is when the divisor is 0.  Edge cases like this should have their behavior defined by the tests